testing/scenario_app/bin/android_integration_tests.dart

```diff
@@ -215,10 +215,6 @@ void main(List<String> args) async {
       await Future.wait(pendingComparisons);
     });
 
-    await step('Flush logcat...', () async {
-      await logcat.flush();
-    });
-
     exit(0);
```

Comment:
This seems not quite right to me - can we instead remove sink from the pipe and then flush?